docs/configuration.adoc

```diff
@@ -242,7 +242,7 @@ kubectl --namespace=default get secret ocirsecret --export -o yaml | kubectl app
 
 == Configure Calico parameters
 
-The calico parameters control the installation of {uri-calico}[Calico] for {uri-calico-policy}[network policy].
+The Calico parameters control the installation of {uri-calico}[Calico] for {uri-calico-policy}[network policy].
```

Comment:
Can you please elaborate a bit more on the configuration options for Calico? e.g. which combination of settings work, flannel/npn. Best if you can put it in the table